be/src/storage/rowset_merger.cpp

```diff
@@ -260,7 +260,9 @@ class RowsetMergerImpl : public RowsetMerger {
         std::vector<ColumnId> sort_key_idxes;
         if (schema.sort_key_idxes().size() > 1) {
             sort_key_idxes = schema.sort_key_idxes();
-            if (!PrimaryKeyEncoder::create_column(schema, &sort_column, sort_key_idxes).ok()) {
+            std::vector<ColumnId> sort_key_iota_idxes(sort_key_idxes.size());
```

Comment:
What is the meaning of iota?

Reply:
Fills the range vector with sequentially increasing values, starting with value and repetitively evaluating ++value.